packages/dev/core/src/scene.ts

```diff
@@ -5223,14 +5229,13 @@ export class Scene extends AbstractScene implements IAnimatable, IClipPlanesHold
         forEach =
             forEach ||
             ((item: any) => {
-                return;
+                return true;

```

Comment:
we don't need to have the redefinition of foreach in that case. If it is null, it will be ignored. So no need to define it in that case